install_config/persistent_storage/storage_classes.adoc

```diff
@@ -277,6 +282,38 @@ parameters:
 <3> storageAccount: Azure storage account name. If storage account is not provided, all storage accounts associated with the resource group are searched to find one that matches skuName and location. If storage account is provided, skuName and location are ignored.
 ====
 
+[[ceph-persistentdisk-cephRBD]]
+=== Ceph RBD
+
+.ceph-storageclass.yaml
+====
+[source,yaml]
```

Comment:
The `provisioner` and `parameters` lines do not need to be indented. It will be recognized as an invalid yaml otherwise.